src/Integration/Suppression/SuppressedIssueMatcher.cs

```diff
@@ -24,7 +24,7 @@
 
 namespace SonarLint.VisualStudio.Integration.Suppression
 {
-    internal class SuppressedIssueMatcher : ISuppressedIssueMatcher
+    public class SuppressedIssueMatcher : ISuppressedIssueMatcher
```

Comment:
This class is in the `Integration` project so it needs to be public to be visible to the `RoslynAnalyzerManager` in the `Integration.Vsix` project.

I'll move some of the non-VS dependent classes to the `Core` project in a separate commit.